checks/hostPortSet.yaml

```diff
@@ -12,4 +12,7 @@ schema:
       items:
         properties:
           hostPort:
-            const: 0
\ No newline at end of file
+            const: 0
+mutations:
+  - op: remove
+    path: /ports[*]/hostPort
```

Comment:
is it possible to make this syntax `/ports/*/hostPort`? That way it is closer to JSON patch.

We could even do something like `strings.Replace("/*/", "[*]/")`